.github/workflows/ci.yml

```diff
@@ -17,12 +17,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby:
-          - 2.6
-          - 2.7
+        ruby: ["2.6", "2.7", "3.0"]
```

Comment:
Just one more thing, while we're at it, you could add 3.1 too since that's out now. 😃

Reply:
It wasn’t available yet last week for github actions, I can try again :sweat_smile:

Reply:
Ah it seems that’s because we’re using `actions/setup-ruby@v1` which is deprecated. Going to update it.

Reply:
Yup, I was about to comment about that! 😃